packages/editor/src/components/block-icon/index.js

```diff
@@ -9,6 +9,16 @@ import classnames from 'classnames';
 import { Dashicon, SVG } from '@wordpress/components';
 import { createElement, Component } from '@wordpress/element';
 
+function isSVGIcon( icon ) {
+	if ( icon && icon.type === 'svg' ) {
+		return true;
+	} else if ( icon && icon.type.name === 'SVG' ) {
```

Comment:
You can check also refactor it as:

```js
return icon && ( icon.type === 'svg' || icon.type === SVG );

Reply:
You mean 

```
return icon && ( icon.type === 'svg' || icon.type.name === SVG );
```

right?

If so, YAY, that's what I wanted! Thank you. Will update.

Reply:
![screen shot 2018-10-24 at 11 40 22](https://user-images.githubusercontent.com/699132/47421800-a0016880-d781-11e8-87b9-37f02728364f.png)

Nope, my code is correct. `.name` is less reliable.

Reply:
Oh really, will that work? Let me give it a shot! Thanks again.

Reply:
it's the same thing as with `<svg`  - type is the name of the component that was used to create an actual element - in first case it's `svg` (string for html tag has special handling), in the second it's the `SVG` component.

Reply:
Okay, pushed https://github.com/WordPress/gutenberg/pull/10987/commits/1f4a19bab1ccb7be1a46dbc36e72a7b6a54b5ef1 — is that kosher? I don't like creating an entirely new function if we don't have to.